belvedere/src/main/java/zendesk/belvedere/ImageStreamService.java

```diff
@@ -30,9 +22,8 @@
     List<MediaResult> queryRecentImages(int count) {
         final List<MediaResult> mediaResults = new ArrayList<>();
 
-        final String order = String.format(Locale.US, "%s DESC LIMIT %s", MediaStore.Images.ImageColumns.DATE_TAKEN, count);
-        final Cursor cursor = context.getContentResolver()
-                .query(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, PROJECTION, null, null, order);
+        final Cursor cursor = new ImageStreamCursorProvider(context, Build.VERSION.SDK_INT)
```

Comment:
Can we pass context as method param rather than constructor so we can make ImageStreamCursorProvider a field & reuse a single instance for multiple calls?